### PR TITLE
[Cookbook][Form] update form_customization.rst

### PR DESCRIPTION
**Pull request by gilles-g**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | 2.3+ |

### CODE REVIEW

**Comment by javiereguiluz** [comment]
@xabbuh @WouterJ any comment on this pull request? Thanks.

File: cookbook/form/form_customization.rst [hunk 1]
```diff
@@ -734,6 +734,24 @@ You can also override the markup for an entire field row using the same method:
             <?php echo $view['form']->widget($form) ?>
         </div>
 
+How to Customize a collection Field
```
Comment (xabbuh) [inline cookbook/form/form_customization.rst]: "Collection" should be uppercased according to our style guide.

**Comment by xabbuh** [comment]
I think we should just handle this in #4629 as suggested by @WouterJ in https://github.com/symfony/symfony-docs/pull/4629#issuecomment-104282997. I will take care of it.

**Comment by WouterJ** [comment]
Hi @spike31!

I've taken your and @xabbuh's PRs and combined them into https://github.com/symfony/symfony-docs/pull/5643. Thanks you for documenting this feature!

**Pull request closed without merging**